package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "revai-node-sdk",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
should this be version `2.2.0`? I believe the last number is bug fixes, whereas the minor number is for non-breaking features